src/lib/repositories/impl/order_repository_impl.py

```diff
@@ -30,5 +32,20 @@ def update_by_id(self, order_id, order):
     def get_orders_to_process(self):
         orders = self.get_all()
 
-        orders_to_process = filter(lambda order: order.status == OrderStatus.NEW_ORDER, orders)
+        orders_to_process = filter(
+            lambda order: order.status == OrderStatus.NEW_ORDER, orders
+        )
         return list(orders_to_process)
+
+    def get_chefs_with_assigned_orders(self, chef_ids):
+
+        orders = self.get_all()
+        assigned_chef_map = reduce(
```

Comment:
Could you be more explicit naming variables and functions?

This variable `assigned_chef_map` holds a key value dict that contains `chef_id` as keys and an array of `assigned orders to the chef` as value. Could you at least try to include the following keywords: `chef, assigned_orders,  map`?

Same for the fuction below `assigned_chef_reducer`, this function reduces an array of chefs to map of chefs to the their assigned orders. Could you be more explicit naming this function?

Reply:
done